packages/prisma/schema.prisma

```diff
@@ -484,6 +486,30 @@ model App {
   ApiKey      ApiKey[]
 }
 
+model App_RoutingForms_Form {
+  id          String                          @id @default(cuid())
+  description String?
+  routes      Json?
+  createdAt   DateTime                        @default(now())
+  updatedAt   DateTime                        @updatedAt
+  name        String
+  fields      Json?
+  user        User                            @relation("routing-form", fields: [userId], references: [id], onDelete: Cascade)
+  userId      Int
+  responses   App_RoutingForms_FormResponse[]
+  disabled    Boolean                         @default(false)
+}
+
+model App_RoutingForms_FormResponse {
+  id           Int                   @id @default(autoincrement())
+  formFillerId String                @default(cuid())
+  form         App_RoutingForms_Form @relation(fields: [formId], references: [id], onDelete: Cascade)
+  formId       String
+  response     Json
+
+  @@unique([formFillerId, formId])
+}
```

Comment:
For MVP is fine but as @agustif said, we should try to make this as generic as possible. Also I don't see any relationship to the actual `App` model. I don't know if this is intended or not.

```suggestion
model App_Form {
  id          String                          @id @default(cuid())
  description String?
  routes      Json?
  createdAt   DateTime                        @default(now())
  updatedAt   DateTime                        @updatedAt
  name        String
  fields      Json?
  user        User                            @relation("routing-form", fields: [userId], references: [id], onDelete: Cascade)
  userId      Int
  app         App                             @relation(fields: [appId], references: [slug], onDelete: Cascade)
  appId       Int
  records     App_Record[]
  disabled    Boolean                         @default(false)
}

model App_Record {
  id           Int                   @id @default(autoincrement())
  formFillerId String                @default(cuid())
  form         App_Form @relation(fields: [formId], references: [id], onDelete: Cascade)
  formId       String
  data         Json

  @@unique([formFillerId, formId])
}
```